Source/Core/Core/Brawlback/BrawlbackUtility.cpp

```diff
@@ -5,15 +5,72 @@
 
 namespace Brawlback
 {
+    void ResetRollbackInfo(RollbackInfo& rollbackInfo)
```

Comment:
Name is better but.. AFAICT this function is not used anywhere. I think it should be removed unless it is actually used